src/database/schema/ChannelListSchema.ts

```diff
@@ -563,7 +564,7 @@ class ChannelList implements BaseDbSchema {
       description: data?.appAdditionalData?.message,
       unreadCount: 0,
       channelType: isGroup ? CHANNEL_GROUP : PM,
-      lastUpdatedAt: data?.channel?.last_message_at,
+      lastUpdatedAt: data?.channel?.last_message_at || data?.channel?.updated_at,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The update to use `updated_at` if `last_message_at` is not available is a good fallback mechanism. It ensures that `lastUpdatedAt` always has a value even when `last_message_at` is null or undefined.
```diff
- lastUpdatedAt: data?.channel?.last_message_at,
+ lastUpdatedAt: data?.channel?.last_message_at || data?.channel?.updated_at,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->